### DISCUSSION TITLE
National forest inventory

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#84] [Planning consideration]
## National forest inventory

All forest and woodland area over 0.5 hectare with a minimum of 20% canopy cover, or the potential to achieve it. Collated by the Forestry Commission

We are looking at this planning consideration because there is a national dataset available and this data helps users to identify areas of woodland, which will be useful in understanding existing land use, habitats, and potentially BNG implications

**Comment by colmjude** [comment]
### Existing dataset

Documentation page: https://data-forestry.opendata.arcgis.com/datasets/83ff06ae0fd34452af9efaddd9d221e8_0/about
Map to explore the data: https://data-forestry.opendata.arcgis.com/datasets/83ff06ae0fd34452af9efaddd9d221e8_0/explore

First published March 2024, last updated April 2024
Records: 360,143
Licence: OGL3

**Reply by colmjude** [reply]
An example record contains a geometry (polygon) and these fields

| field | value |
| --- | --- |
| OBJECTID | 520230 |
| CATEGORY | Woodland |
| IFT_IOA | Conifer |
| COUNTRY | England |

**Reply by colmjude** [reply]
IFT = interpreted forest type. And is done by analysing aerial photography and satellite imagery.

Values are:
Conifer, Broadleaved, Mixed mainly conifer, Mixed mainly broadleaved, Coppice, Coppice with standards, Shrub, Young trees, Felled, Ground prep, Cloud \ shadow, Uncertain, Low density, Assumed woodland, Failed, Windblow.

**Reply by colmjude** [reply]
We have created a [forest-inventory](https://github.com/digital-land/specification/blob/main/content/dataset/forest-inventory.md?plain=1) schema.

We have chosen that name to mirror the name of the dataset closely but we are contemplating renaming it `forest-inventory-area` because the dataset is a set of areas. We are interested in knowing if anyone else has an opinion on this.

| our field name | mapped from | default value |
| --- | --- | --- |
| reference | OBJECTID | |
| name | | |
| organisation | | government-organisation:D85 |
| forest-inventory-category | CATEGORY | |
| interpreted-forest-type | IFT_IOA | |

Although we are not sure `OBJECTID` is suitable to use for the reference. We seen datasets where the OBJECTID changes everytime the dataset is published/updated

**Reply by psd** [reply]
I don't think objectid is a suitable reference, especially as there are several versions of this dataset with different objectid values for each entity. Maybe we should use the entity number as the reference, and create lookups with the resource/objectid as the lookup.

**Reply by psd** [reply]
There might be a reference for each of these shapes we can get from the Forestry Commission

**Reply by eveleighoj** [reply]
How do we manage the lookups for future updates? 

We'd need to agree a process for assigning new resource/objectID lookups for each resource. This would likely need to be automated to not put more pressure on the data. the process would also need to automatically assign new entities when they pop up.

**Reply by psd** [reply]
I think we'd need better ways of matching to existing entities where the geometry is the same, so we'd only need to look at matching or assigning new entities to rows with a new boundary.

**Reply by eveleighoj** [reply]
Matching on geometries is possible however it does introduce some uncertainty and it would be very expensive to do each time a new resource was downloaded. We would need to be fairly careful about how we choose the matching method and would need to communicate the risk.

I definitely think it's worth talking to the Forestry Commission to understand if they have any unique identifiers for this dataset. Even if they don't it'll be good to understand why.

**Reply by colmjude** [reply]
Expensive computationally or people?

Also, is the risk that we might miss match geometries? And if so, what would that mean?

Agree it would be good to speak to Forestry Commission. We need to find the right contact.

**Comment by psd** [comment]
This is just for resources where we don't have a reference, and I imagined the comparison being as simple as querying the existing fact table for the geometry WKT string which can be indexed or hashed.

**Reply by eveleighoj** [reply]
this could work by making the reference column the hash of the geometry. The downside is if the geometries change at all then it won't be able to follow the changes.
